tools/nanopolish/nanopolish_index_variants.xml

```diff
@@ -0,0 +1,89 @@
+<tool id="nanopolish_index_variants" name="Finds SNPs and polishes" version="0.1.0">
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+        <!-- first index then polish -->
+    <command detect_errors="exit_code"><![CDATA[
+
+        cp $input_merged reads.fasta && cp $input_reads_tar fast5_files.tar.gz && tar -xzf fast5_files.tar.gz &&
+        nanopolish index -d fast5_files/ reads.fasta && ls -all . &&
+
+        cp $input_reads_bam reads.bam && cp $input_reads_bai reads.bam.bai &&
+        nanopolish variants --consensus polished.fa -w "tig00000001:200000-202000" -r reads.fasta -b reads.bam -g $input_genome_fa
+    ]]></command>
+    <inputs>
+      <!-- index inputs -->
+        <param type="data" name="input_merged" format="fasta" />
+        <param type="data" name="input_reads_tar" format="fast5.tar.gz" />
+
+      <!-- variants consensus inputs -->
+        <param type="data" name="input_reads_bam" format="bam" />
+        <param type="data" name="input_reads_bai" format="bai" />
```

Comment:
bai files are automatically generated by Galaxy as part of the metadata, so you don't need to take care of it.

Reply:
Planemo fails once bai param is removed

Reply:
```
Error: could not load the .bai index file for reads.bam
Please run 'samtools index reads.bam' before nanopolish
```

Reply:
https://github.com/galaxyproject/tools-iuc/blob/411130b45dc30f7f24f41cdeec5e148c5d8faf40/tool_collections/samtools/samtools_bedcov/samtools_bedcov.xml#L11

this example is for multiple BAM files, you probably just need the `ln -s` lines